irc_bot/simple_irc_bot.py

```diff
@@ -444,7 +463,7 @@ def identify_with_nickserv(self):
 
             # Identify with NickServ
             log.info('Identifying with NickServ as %s', self.nickname)
-            self.send_privmsg('NickServ', 'IDENTIFY %s' % self.nickserv_password)
+            self.send_privmsg('NickServ', 'IDENTIFY %s %s' % (self.nickname, self.nickserv_password))
```

Comment:
I did this while debugging. I can undo it, it shouldn't be necessary since `self.nick` is called earlier. Doesn't hurt